.eslintrc

```diff
@@ -51,6 +52,7 @@ rules:
   no-var: 2
   object-curly-spacing: [2, never]
   one-var: [2, never]
+  operator-linebreak: [2, "after"]
```

Comment:
I would rather keep the style consistent and do no use the `overrides` option.
What do we do here :grin:?